docs/api/ref/Errors.md

```diff
@@ -135,8 +143,34 @@ enum ResultError {
 
 Contains constants representing the errors that can be encountered when calling the [`getResult`](DesktopAgent#findintent) method on the [IntentResolution](Metadata#intentresolution) Object.
 
-#### See also
+**See also:**
 
 * [`DesktopAgent.addIntentListener`](DesktopAgent#addintentlistener)
 * [`DesktopAgent.raiseIntent`](DesktopAgent#raiseintent)
 * [`IntentResolution`](Metadata#intentresolution)
+
+## `BridgingError`
```

Comment:
Should this type be here or under the bridging section?

Reply:
It turns out that we do need to return these through the API if the failure of the API call was entirely related to a problem with bridging (e.g. it targeted a remote Desktop Agent that has gone away, the connection to the bridge for this agent has dropped etc.). Hence, I've gone the other way and added a note on those API calls that might need to return these error messages (API calls with non-trivial responses).